designing-reports/report-designer-tools/web-report-designer/report-templates.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Report Templates
+page_title: Report Templates in Web Report Designer
+description: Learn more about Report Templates in the Telerik Reporting Web Report Designer, how to create and reuse them.
+slug: web-report-designer-report-templates
+tags: web, report, designer, wrd, report, template, trtx
+published: True 
+position: 1 
+---
+
+<style>
+img[alt$="><"] {
+  border: 1px solid lightgrey;
+}
+</style>
+
```

Comment:
How many personas is this article targeting? Is it a single persona? If not, make sure to split the content for the different personas into independent articles.